### PR TITLE
More TS yet (for #868)

### PR DESCRIPTION
**Pull request by baltpeter**

### CODE REVIEW

File: src/Components/Accordion.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import type { ComponentChildren } from 'preact';
+import { useState } from 'preact/hooks';
+
+type AccordionProps = {
+    id: string;
+    title: string;
+    expanded: boolean;
```
Comment (baltpeter) [inline src/Components/Accordion.tsx:7]: We should rename that to `expandedInitially` or something like that.

Comment (zner0L) [reply src/Components/Accordion.tsx:7]: Yes, please do that.

**Review by zner0L — DISMISSED** [review]
If you don't plan on implementing your own suggestion, this is ready to be merged.

**Comment by baltpeter** [comment]
We'll have to remember the rename when merging #874.

**Review by zner0L — APPROVED** [review]

**Pull request merged**